docker/prod/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG IMAGE=node:18-alpine
+ARG IMAGE=node:16.10.0-alpine
```

Comment:
However, I am not sure if we should keep v18 and install npm v7.

Reply:
I'm leaning towards keeping 16.10 as this is what we have in the tests: https://github.com/energywebfoundation/ssi-hub/blob/744754e73dfc9f2b835b4202a3e910602ca566f0/.github/workflows/build-test.yml#L67